sdk/python/cmd/pulumi-language-python/main.go

```diff
@@ -96,6 +96,7 @@ func main() {
 	flag.StringVar(&tracing, "tracing", "", "Emit tracing to a Zipkin-compatible tracing endpoint")
 	flag.String("virtualenv", "", "[obsolete] Virtual environment path to use")
 	flag.String("root", "", "[obsolete] Project root path to use")
+	flag.String("mypy", "", "[obsolete] Use mypy to type check")
```

Comment:
Could we expose this in a way that'd allow multiple different type checkers? For example, if I wanted to use [pyright](https://github.com/Microsoft/pyright) rather than mypy?

Reply:
Could do. Something like `"typechecker": "mypy"`, and just support "mypy" and "pyright" to start with.